### DISCUSSION TITLE
Update configuration variables without admin permission?

### DISCUSSION THREAD

**Discussion by MikeMatusz** [community/community#43962] [Actions]
### Select Topic Area

Product Feedback

### Body

I was excited about the introduction of configuration variables, as there are several non-secret outputs I want to store after running a TerraForm apply (like an S3 bucket name and region), and use in various workflows without re-running TF unless the infrastructure configuration changes.  The most inconvenient part about using Secrets for this was having to create and continually renew a PAT for this.  I had been expecting that configuration variables improve this situation, but as they still require admin privileges to update, the only advantage is really that you can see what their values are without going back to the workflow log of the run that created them.

Is there a better way to story this kind of non-secret at the repository level (or at least accessible across all workflow runs) that wouldn't require maintaining a PAT?  I've considered the Actions cache, but the rules on the visibility of that across branches and what to do in the event of a cache miss don't really seem suited to this use case.  I could potentially have the action commit them to a file and read them from there, but it definitely feels like something is missing from Actions to support this.  If I don't need to rebuild my infrastructure, I just want to know what the most recent values are to push my content to the bucket that exists.

Would it be possible to allow configuration of individual variables to be editable using the `GITHUB_TOKEN` used by the Actions or something like that?  Or is there another mechanism I'm missing entirely.

**Comment by Fb9027** [comment]
Yes

**Reply by Fb9027** [reply]
I have red understand it's good~~~~
